activerecord/test/cases/cache_key_test.rb

```diff
@@ -123,9 +123,10 @@ class CacheMeWithVersion < ActiveRecord::Base
     test "updated_at on class but not on instance raises an error" do
       record = CacheMeWithVersion.create
       record_from_db = CacheMeWithVersion.where(id: record.id).select(:id).first
-      assert_raises(ActiveModel::MissingAttributeError) do
+      error = assert_raises(ActiveModel::MissingAttributeError) do
```

Comment:
We can use `assert_raises(*exp, match: nil, &block)` method if you like that has been introduced via https://github.com/rails/rails/pull/46611/files

Reply:
Ah nice! I'll make the change.

Reply:
@yahonda I’ve made the change.